cmd/tailscale/cli/cli.go

```diff
@@ -75,6 +75,8 @@ func CleanUpArgs(args []string) []string {
 	return out
 }
 
+const tsUsage = "tailscale "
```

Comment:
I think I'd prefer to just inline this everywhere and add a test via the walkSubcommands thingy that nobody forgets it

Reply:
Yeah, fair. I experimented with showing `tailscale [--socket <sockfile>] <subcmd> [subcmd flags] on all commands but it was too much. I'll just hard-code these prefixes for now.